man/md2man-all.sh

```diff
@@ -9,6 +9,12 @@ cd "$(dirname "$(readlink -f "$BASH_SOURCE")")"
 	pwd
 }
 
+has_md2man=`which go-md2man > /dev/null 2>&1 && echo yes || echo no`
+if [ $has_md2man == "no" ]; then
```

Comment:
I'd prefer something more like this:

```
if ( which go-md2man &>/dev/null ); then
  # ...
fi
```

As it's cleaner.

Reply:
Hi @cyphar  😄 

Thanks for your advice, I changed it just now !